### ISSUE TITLE
Workspace names on workspace switcher?

### ISSUE THREAD

**Issue by davehayes** [gmc-holle/xfdashboard#105]
Is it possible to add workspace names onto the workspace switcher so you can see which name you are switching to? I looked at themes but it's currently opaque to me how to do that.

**Comment by gmc-holle** [comment]
Yes, it should be possible but it requires a change to the code as the workspace switcher does not support workspace names yet. So you cannot change it just with a theme.

I'll mark this issue as enhancement and try to implement it with the next versions. I'll keep you updated.

**Comment by gmc-holle** [comment]
The feature was implemented with commit bce77d52c007ebe304248baccabcf90a6da61b7a and will be included in next release.

By default the workspace's name will not be shown in the workspace selector to match the current behaviour. You have to enable it in your theme. You can used this CSS snippet:

```
XfdashboardLiveWorkspace
{
    show-workspace-name: true;
    workspace-name-padding: 4.0; /* Gap in pixels between workspace name and buttom of workspace actor */
}

XfdashboardLiveWorkspace .title
{
    /* Style the workspace name actor here like XfdashboardLiveWindow .title */
}
```

**Comment by davehayes** [comment]
Thank you! :)
